src/main/java/io/kamax/matrix/hs/_MatrixRoom.java

```diff
@@ -47,6 +48,8 @@
 
     Optional<_MatrixContent> getAvatar();
 
+    String getId();
```

Comment:
What's the relation with this PR?

Reply:
See above.

Reply:
I gave the reason for this in the wrong comment. I need the ID for my test cases. My question regarding moving code to another PR should have been placed here. So should I move this part to another PR?

Reply:
It's fine, you can leave it if it's needed for tests - I'm even surprised it wasn't needed until now!